connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnectorTest.java

```diff
@@ -59,12 +59,16 @@ public void testMirrorCheckpointConnectorDisabled() {
 
         Set<String> knownConsumerGroups = new HashSet<>();
         knownConsumerGroups.add(CONSUMER_GROUP);
+
+        // MirrorCheckpointConnector as minimum to run taskConfig()
+        // expect no task will be created
+        assertEquals(0, new MirrorCheckpointConnector(knownConsumerGroups, config).taskConfigs(1).size(), "MirrorCheckpointConnector not disabled");
+    }
+
+    private void assertMirrorCheckpointConnectorDisabled(MirrorCheckpointConnector connector) {
```

Comment:
Is this change still necessary?

Reply:
I thought I removed it last time! should be removed now